packit_service/service/api/healthz.py

```diff
@@ -0,0 +1,42 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from logging import getLogger
+
+from flask import jsonify
+from flask_restplus import Namespace, Resource
+
+logger = getLogger("packit_service")
+
+ns = Namespace("healthz", description="Health checks")
+
+
+@ns.route("/")
+class HealthCheck(Resource):
+    def get(self):
+        return jsonify({"msg": "We are healthy!"})
+
+    def head(self):
+        return self.get()
+
+    def post(self):
```

Comment:
I understand head and get, but post? I'd say it's fine if the webserver responds with != 200 for post on /healthz

Reply:
Yeah, I don't understand it either :-D  https://github.com/packit-service/packit-service/commit/5937c5ba056f3a97970935e3f953acaa32a15bd3

Reply:
:D

Reply:
no idea what I was thinking back then